internal/interfaces/grpc/handler/operator.go

```diff
@@ -987,8 +987,11 @@ func parsePrice(p *pbtypes.Price) (price application.Price, err error) {
 		basePrice = decimal.NewFromFloat32(p.GetBasePrice())
 		quotePrice = decimal.NewFromFloat32(p.GetQuotePrice())
 	}
-	pp := application.Price{basePrice, quotePrice}
-	if err = price.Validate(); err != nil {
+	pp := application.Price{
+		BasePrice:  basePrice,
+		QuotePrice: quotePrice,
+	}
+	if err = pp.Validate(); err != nil {
```

Comment:
🤔

Reply:
if we pass a price like 0.001 will still return (wrongly) the error

This check has been introduced in latest rc, is the issue
https://github.com/tdex-network/tdex-daemon/blob/master/internal/core/application/types.go#L127

Reply:
https://github.com/tdex-network/tdex-daemon/issues/456

Reply:
no 0.001 is not lower than 0 and this is the right fix: the problem was that we were validating `price`, which is the return value of the `parsePrice` function, instead of validating `pp` which is the price of the incoming request.

Reply:
🙏  you right